tasks/version.rb

```diff
@@ -39,3 +39,27 @@
     end
   end
 end
+
+task :expeditor_update_version do
```

Comment:
i don't know if this is a stub, but it doesn't seem to be useful, and it seems like the caller should be responsible for swallowing/ignoring exceptions directly.